### ISSUE TITLE
app.terraform.io returns error to CLI: Terraform version "0.12.0-rc1" is not a valid version

### ISSUE THREAD

**Issue by fbdntm** [hashicorp/terraform#21306]

**Comment by apparentlymart** [comment]
Hi @fbdntm! Thanks for reporting this.

Unfortunately since you deleted the entire [issue template](https://raw.githubusercontent.com/hashicorp/terraform/master/.github/ISSUE_TEMPLATE/bug_report.md) we can't really tell what you were doing when this occurred, so I'm not sure what part of the system this problem relates to.

If you could edit your report to include the relevant information from the issue template, that'd allow us to triage the problem to the right subsystem and team. Thanks!

**Comment by fbdntm** [comment]
### Terraform Version
```
Terraform v0.12.0-rc1
```

### Terraform Configuration Files

```hcl
terraform {
  backend "remote" {
    organization = "XXX"
    workspaces {
      name = "Test"
    }
  }
}
```

### Expected Behavior
Workspace Test created on app.terraform.io

### Actual Behavior
Initializing the backend...
Error loading state: Error creating workspace Test: Invalid Attribute
Terraform version "0.12.0-rc1" is not a valid version

### Steps to Reproduce
terraform init

### Additional Context
Using Terraform Enterprise Free Tier

### References
None

**Comment by apparentlymart** [comment]
Thanks for that extra context, @fbdntm!

Since you mentioned the Enterprise Free Tier, it seems like this problem had something to do with the state storage feature, so I'm going to pass this on to the team that is working on the feature.

Terraform Enterprise usually doesn't support prerelease versions of Terraform, so this _may_ be expected but we'll check to make sure that this isn't a bug that would also affect the final release.

**Comment by fbdntm** [comment]
Thank you @apparentlymart.
The following link mentions the Free Tier to be available in the "newest 0.12 preview releases":
https://www.terraform.io/docs/enterprise/free/index.html#upgrade-to-terraform-0-11-13-or-later

**Comment by teamterraform** [comment]
Hi @fbdntm!

Since v0.12.0 final was released some time ago now and thus there's no reason to be using its release candidates any longer, we're going to close this out now. Thanks for reporting it!
